cluster/juju/layers/kubeapi-load-balancer/layer.yaml

```diff
@@ -1,9 +1,12 @@
+repo: https://github.com/kubernetes/kubernetes.git
 includes:
-  - layer:nginx
-  - layer:tls-client
-  - interface:public-address
-repo: https://github.com/juju-solutions/kubernetes-load-balancer
+  - 'layer:nginx'
+  - 'layer:tls-client'
+  - 'interface:public-address'
 options:
+  basic:
+    packages:
+      - 'nfs-common'
```

Comment:
The load balancer doesn't need nfs-common, as its not running the kubelet workers. Therefore its not loading any nfs volumes on behalf of the cluster.